Sources/App/Models/SocialAccount/Social.swift

```diff
@@ -0,0 +1,86 @@
+import Vapor
+import FluentProvider
+import HTTP
+
+final class Social: Model {
+  let storage = Storage()
+  
+  var socialId: Int
```

Comment:
Interesting question. In Swift we have `camelCase`. But in the most server frameworks it's considered to use `under_score`.

So, we have Server-Side-Swift. What's better to use? 🤔

Reply:
In code of course camelCase, but all keys and so would be transform to server format